src/e2e/iOS/helpers/gesture.ts

```diff
@@ -25,18 +33,17 @@ const gesture = async (
   }, [])
 
   // add first and last action
-  const actions = [
-    { action: 'press', x: xStart, y: yStart },
-    ...moveActions,
-    { action: 'release' },
-    WAIT_ACTION,
-  ] as TouchAction[]
+  const actions = [{ action: 'press', x: xStart, y: yStart }, ...moveActions, { action: 'release' }] as TouchAction[]
 
   // webdriverio has some problem about type for TouchAction[] that is why we add @ts-ignore
   // We didn't use touchPerform here because webdriverio calls touchPerform in the background. https://github.com/webdriverio/webdriverio/blob/aea3d797ab1970309a60c43629f74154453597e9/packages/webdriverio/src/commands/constant.ts#L100
   // eslint-disable-next-line @typescript-eslint/ban-ts-comment
   // @ts-ignore
   await browser.touchAction(actions)
+
+  if (waitForEditableAfterGesture !== undefined) {
+    await waitForEditable(browser, waitForEditableAfterGesture)
+  }
```

Comment:
I would avoid doing this here. It's not really related to the gesture itself. The consuming code can call `waitForEditable` afterwards. It's about the same amount of code as specifying the option.

If you'd like to provide an alias for the new thought gesture + wait + edit, you could create a `newThought` helper which executes all three, like we do for Puppeteer:

https://github.com/cybersemics/em/blob/3dcad97b2e7f1b1150218f60095492a6433ced18/src/e2e/puppeteer/helpers/newThought.ts#L4-L12